livox_ros2_driver/livox_ros2_driver/lddc.cpp

```diff
@@ -746,11 +747,17 @@ void Lddc::checkTemperature(diagnostic_updater::DiagnosticStatusWrapper & stat)
       level = DiagStatus::ERROR;
     }
 
+    if (level != DiagStatus::OK) {
+      if (error_str != "") error_str += ", ";
+      error_str += broadcast_code + ": " + temperature_dict_.at(level);
+    }
+
     stat.add(broadcast_code, temperature_dict_.at(level));
     whole_level = std::max(whole_level, level);
   }
 
-  stat.summary(whole_level, temperature_dict_.at(whole_level));
+  if (whole_level == DiagStatus::OK) error_str = temperature_dict_.at(whole_level);
```

Comment:
I don't think we need `dict` here. How about just writing `error_str = "OK";` or something?

Reply:
https://github.com/tier4/livox_ros2_driver/pull/14/commits/5d7c97ba3f8107c9c915d182e6e4933d01b8a2c6

Reply:
sorry dit not insert

Reply:
I fixed